src/analysis/special_functions/trigonometric/angle.lean

```diff
@@ -71,6 +71,18 @@ begin
   simp [two_mul, sub_eq_add_neg]
 end
 
+@[simp] lemma two_nsmul_pi_div_two : (2 : ℕ) • (↑(π / 2) : angle) = π :=
```

Comment:
Does this generalize to `(2 : R) •`?

Reply:
`angle` doesn't have scalar actions by arbitrary rings; you can only multiply angles by integers.

Reply:
This is true with `π` replaced with any real, right? So maybe
```suggestion
@[simp] lemma two_nsmul_coe_div_two (x : ℝ)  : (2 : ℕ) • (↑(x / 2) : angle) = x :=
```
combined with a lemma that says `-(↑π : angle) = π` or similar.

Reply:
I've made that change for `two_nsmul_pi_div_two` and `two_zsmul_pi_div_two`, and updated the proof of `two_nsmul_neg_pi_div_two` to use `two_nsmul_coe_div_two`.